### PR TITLE
BookA-Pset_InSituTestLugeon

### PR DESCRIPTION
**Pull request by larswik**
# Pset: Pset_InSituTestLugeon

## Description: 

Properties for in-situ lugeon tests

| Property | Datatype | Description |
|----------|----------|-------------|
|LugeonValue|IfcReal|Lugeon Value = (q / L) x (P0 / P) q - flow rate [lit/min] L - Length of the borehole test interval [m] P0 - reference pressure of 1 MPa [MPa] P - Test pressure [MPa]|
|LimitPressure|IfcPressureMeasure|The quality of the soil that enables water to move through the profile. Permeability is measured as the number of inches per hour that water moves through the saturated soil. Terms describing permeability are as follows|

### CODE REVIEW

File: IFC4x3/Properties/l/LimitPressure_0i0VHvsiT9wBPIBgvQ3lXi/Documentation.md [hunk 1]
```diff
@@ -0,0 +1 @@
+The quality of the soil that enables water to move through the profile. Permeability is measured as the number of inches per hour that water moves through the saturated soil. Terms describing permeability are as follows
```
Comment (SergejMuhic) [inline IFC4x3/Properties/l/LimitPressure_0i0VHvsiT9wBPIBgvQ3lXi/Documentation.md:1]: Something is of in this description and it seems not complete.

File: IFC4x3/Properties/l/LugeonValue_3KqDg25zjESeuHw4dJBr66/Documentation.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Lugeon Value = (q / L) x (P0 / P) q - flow rate [lit/min] L - Length of the borehole test interval [m] P0 - reference pressure of 1 MPa [MPa] P - Test pressure [MPa]
```
Comment (SergejMuhic) [inline IFC4x3/Properties/l/LugeonValue_3KqDg25zjESeuHw4dJBr66/Documentation.md]:
```suggestion
Lugeon Value = (q / L) x (P0 / P)
q - flow rate [lit/min]
L - Length of the borehole test interval [m]
P0 - reference pressure of 1 MPa [MPa]
P - Test pressure [MPa]
```

File: IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/PropertySets/Pset_InSituTestLugeon/Documentation.md [hunk 3]
```diff
@@ -0,0 +1 @@
+Properties for in-situ lugeon tests
```
Comment (SergejMuhic) [inline IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/PropertySets/Pset_InSituTestLugeon/Documentation.md]:
```suggestion
Properties for in-situ lugeon tests.
```

File: IFC4x3/Properties/l/LimitPressure_0i0VHvsiT9wBPIBgvQ3lXi/Documentation.md (hunk 1 above)
Comment (larswik) [reply IFC4x3/Properties/l/LimitPressure_0i0VHvsiT9wBPIBgvQ3lXi/Documentation.md:1]: Most definitely! @j-weil.

**Comment by larswik** [comment]
This Pset and Pset_InSituTestPumping were merged into Pset_InSituTestHydro. Kept PR #610 and renamed that Pset.

**Pull request closed without merging**